assets/armsnippets.jsonc

```diff
@@ -838,15 +841,13 @@
             "    \"name\": \"[concat('${1:account-name}', '/gremlin/', '${2:database-name}')]\",",
             "    \"apiVersion\": \"2016-03-31\",",
             "    \"dependsOn\": [",
-            "    \"[resourceId('Microsoft.DocumentDB/databaseAccounts/', '${1:account-name}')]\"",
+            "       \"[resourceId('Microsoft.DocumentDB/databaseAccounts/', '${1:account-name}')]\"",
             "    ],",
             "    \"properties\": {",
             "        \"resource\": {",
             "            \"id\": \"${2:database-name}\"",
             "        },",
-            "        \"options\": {",
-            "            \"throughput\": ${3:400}",
-            "        }",
+            "        \"options\": {}",

```

Comment:
Throughput can be quite expensive for cosmos, I think we should leave that here.  Or was there a specific reason you left it out?

Reply:
It was not validating as it was. I will need to dig into the resource, deploy a few, and wrap my head around how this should be configured.

Reply:
Did you try with quotes around it?  Sometimes the schemas are wrong about the types of things (or maybe its max is too large for an integer).  See https://github.com/Azure/azure-quickstart-templates/blob/master/101-cosmosdb-gremlin/azuredeploy.json.

Reply:
Yes, quotes fixes it. My conclusion is a schema issue. I am in the process of putting this back and then will follow up with an issue on the schema.

Reply:
Fixed